### ISSUE TITLE
Cannot detect disabled menu items

### ISSUE THREAD

**Issue by bjhartin** [Hammerspoon/hammerspoon#1120]
The AXEnabled value for menu items is returning a table, e.g. `{"cmd", "shift"}` instead of a boolean as the [docs](http://www.hammerspoon.org/docs/hs.application.html#getMenuItems) describe.

Here is how to reproduce this:

```lua
> current = hs.application.frontmostApplication()

> menu = hs.application.getMenuItems(current)

> hs.inspect(menu[1].AXEnabled)

{ "cmd", "shift" }

**Comment by latenitefilms** [comment]
FYI: I can also reproduce this bug.

**Comment by cmsj** [comment]
Will be fixed in 0.9.51 :)

**Comment by bjhartin** [comment]
💯
